examples/blogdown.yaml

```diff
@@ -12,27 +12,10 @@ jobs:
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v2
-
-      - name: Setup R
-        uses: r-lib/actions/setup-r@v1
-
-      - name: Install pandoc
-        run: |
-          brew install pandoc
-
-      - name: Cache Renv packages
-        uses: actions/cache@v2
-        with:
-          path: $HOME/.local/share/renv
-          key: r-${{ hashFiles('renv.lock') }}
-          restore-keys: r-
-
-      - name: Install packages
-        run: |
-          R -e 'install.packages("renv")'
-          R -e 'renv::restore()'
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@v1
+      - uses: r-lib/actions/setup-r-dependencies@master
+      - uses: r-lib/actions/setup-pandoc@v1
```

Comment:
Is there a reason you are installing pandoc after the dependencies / R itself?

I think there could potentially be install-time dependencies on pandoc that would fail unless it is installed before R packages

Reply:
No, just didn't think about it. The only thing I care about is that the steps are in the same order in all of the examples.